tests/util/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BACKEND=${1:-"linode:"}
+ITERATIONS=${2:-10}
+OUTPUT_FILE=${3:-"${PWD}/benchmark.out"}
```

Comment:
It seemed to work, the .out files were written correctly in the current directory, but done anyway for consistency.